config/rbac/metrics_role.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Role
```

Comment:
(question/doubt) I still wonder why we cannot use https://github.com/RamenDR/ramen/blob/main/config/rbac/auth_proxy_client_clusterrole.yaml instead.

Reply:
IOW, if we do not need the above file we should remove it?

Reply:
there are errors in prometheus-k8s pods when there is are no permissions. 
```
* failed to list *v1.Service: services is forbidden
* failed to list *v1.Pod: pods is forbidden
* failed to list *v1.Endpoints: endpoints is forbidden
```